instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/DbResponseStatusUtil.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.incubator.semconv.db;
+
+import javax.annotation.Nullable;
+
+/** Utility class for database response status. */
+public class DbResponseStatusUtil {
+  private DbResponseStatusUtil() {}
+
+  @Nullable
+  public static String httpStatusToResponseStatus(int httpStatus) {
+    return httpStatus >= 400 && httpStatus < 600 ? Integer.toString(httpStatus) : null;
+  }
+}
```

Comment:
I think I'd prefer the duplication of copying this method into the three classes where it's used, compared to the extra public api surface area here